app/views/medications/new.html.erb

```diff
@@ -1,8 +1,11 @@
 <p>Enter brand medication name</p>
 <p style="color:red"><%= render partial: '/partials/flash' %></p>
 <section class="med_search_form">
-  <%= form_tag "/medications/search", method: :get %>
-  <%= label_tag :brand_name %>
-  <%= text_field_tag :brand_name %>
-  <%= submit_tag "Find Medication" %>
+  <%= form_tag "/medications/search", method: :get do %>
+    <%= label_tag :brand_name, 'Brand Name' %>
+    <%= text_field_tag :brand_name %>
+    <%= submit_tag "Find Medication" %>
+  <% end %>
```

Comment:
ahh, I'm realizing that since there's a dashboard button in the nav bar having a dashboard button here is redundant. not sure if I maybe just wrote the card description incorrectly when i jotted this idea down, but I was thinking it would make it more user friendly if, when a user is on the search results page and they don't get the results they're looking for, they can easily go back to the search bar page to re-enter their medication search term

Reply:
i'm cool with it either way, we can always remove later if we want. Don't want to hold up this PR :)